build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Build docker script
+set -e
+
+# Enable docker experimental feature
+export DOCKER_CLI_EXPERIMENTAL=enabled
+
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+
+# Build x86 images
+docker build --build-arg IMAGEARCH= \
+                   --build-arg QEMUARCH="x86_64" \
+                   --file Dockerfile --tag "sgerrand/glibc-builder:${CIRCLE_SHA1}-amd64" .
```

Comment:
If you're concerned about shell expansion then suggest localising it and only using one method (e.g. `{}` _or_ `""`, not both).
```suggestion
                   --file Dockerfile --tag sgerrand/glibc-builder:"$CIRCLE_SHA1"-amd64 .
```